### PR TITLE
change logging file descriptor

### PR DESCRIPTION
**Pull request by daedalus**
change the logging file descriptor from a global variable to a class member.

### CODE REVIEW

File: FACTOR.py [hunk 1]
```diff
@@ -654,12 +654,18 @@ def mine():
     global SCRIPTPUBKEY
     if SCRIPTPUBKEY == None:
         SCRIPTPUBKEY = sys.argv[1].strip()
+        
+    os.system("logs")
```
Comment (ReverseControl) [inline FACTOR.py:658]: What does this line do?

Comment (daedalus) [reply FACTOR.py]: hi, forgot an mkdir...

File: FACTOR.py [hunk 2]
```diff
@@ -654,12 +654,18 @@ def mine():
     global SCRIPTPUBKEY
     if SCRIPTPUBKEY == None:
         SCRIPTPUBKEY = sys.argv[1].strip()
+        
+    os.system("mkdir logs")
```
Comment (ReverseControl) [inline FACTOR.py:658]: Should be ``mkdir -p logs`` so that if the directory already exists, it does not give an error.

Comment (daedalus) [reply FACTOR.py]: you are right

**Pull request merged**